### ISSUE TITLE
Parser warnings (subqueries)

### ISSUE THREAD

**Issue by 2busy** [phpmyadmin/phpmyadmin#11602]
When using latest release of XAMPP stack v5.6.14 a bug exists opening User Accounts with phpMyAdmin Version  4.5.1 and  XAMPP stack v5.6.14 has Server version: 10.0.17-MariaDB.  At times the bug does not appear other times an analysis finds 5 unexpected closing brackets which precludes you from getting to the User accounts that reside in Table User in the MySQL database, which is a serious problem. A screen shot is attached.
![user accounts error](https://cloud.githubusercontent.com/assets/15304335/10713154/c88ce3d4-7a77-11e5-9cf2-0bc3603a2c9f.png)

**Comment by lem9** [comment]
Not sure about the parser messages, but it looks like one of your tables in the "mysql" database is corrupted. 

Are you able to reproduce the problem on https://www.phpmyadmin.net/try/ by choosing the stable release, then the MariaDB server?

**Comment by 2busy** [comment]
I'm very much a beginner and greatly appreciate lem9 so quickly commenting on my matter; since as I understand it, you, based on the Home page of phpMyAdmin, are one of the developers of phpMyAdmin and world renowned author on the subject.  It is an honor.

As to a stable release of MariaDB server, I have not switched-out my XAMPP stack with a prior version.   Again, a beginner, not sure how I could do that.

It appears issue #11599 is possibly having a related bug since he is in the Privilege area and in his SQL query it pertains to table db of database MySQL.

Therefore, being of very limited knowledge, it seems the table db in the MariaDB 10.0.17 is corrupted and Apache Friends, MariaDB or someone needs to fix the corrupted table, however that is accomplished.

**Comment by lem9** [comment]
The link I gave above points to our demo server, where you can try phpMyAdmin 4.5.1 (the latest stable version) with a MariaDB server. As you can see on this demo server (using root and no password), there is no problem opening the User accounts page. In phpMyAdmin, open the "mysql" database, then tick the checkbox next to "db" and at the bottom, With selected > Repair table.

**Comment by 2busy** [comment]
lem9,
Thanks for your instructions as to Repair table.  I did the repair and attached the results, it, I believe, was a failed operation.

I did visit the demo and it does, of course, not have issues.  I also noticed the demo MariaDB version is 10.0.21 whereas the current stacks for download have an older version of 10.0.17 (which is the version I could not repair).  .17 is likely fine, and possibly somewhere in the download/install my Dell laptop Windows 8.1 somehow corrupted it.  Looking at MariaDB site, .21 was released 2015-08-06 whereas .17 was released 2015-2-27.

Should you have any resources or links how I whack 10.0.17 and drop .21 in my XAMPP stack, and I am a beginner, I would be appreciative.  I'm off to Google the topic.

Thanks again for all the assistance.
![repair db](https://cloud.githubusercontent.com/assets/15304335/10717608/7822d214-7b2b-11e5-9101-f6f7f93a4b70.png)

**Comment by 2busy** [comment]
I found the same problem on the Apache Friends Forum https://community.apachefriends.org/f/viewtopic.php?f=16&t=72035 that I have been discussing here.  The 5 unexpected closing bracket bug from the suspected corrupt db table in MySQL database when attempting to access User accounts with MariaDB 10.0.17.

Here is a screen shot of the Apache Friends Forum:

![appache friends](https://cloud.githubusercontent.com/assets/15304335/10718666/3f3af7a4-7b46-11e5-9df4-e59a2056c3a5.png)

**Comment by lem9** [comment]
If you reinstall the same XAMPP version, is the table already corrupted?

**Comment by 2busy** [comment]
Yes.  I have reinstalled XAMPP version 5.6.14 (the most recent version) at least 3 or 4 times and each time the table is already corrupted.  I discovered in the Apache Friends Support Forum user linuxs has the same issue thus I registered as coder1 to reply to his thread https://community.apachefriends.org/f/viewtopic.php?f=16&t=72035 that I have the same issue and provided his screen shot in this issue thread above.  Don't know if noting the problem in Apache Friends Support Forum will do any good.

And, in your demo, as I noted above, you installed MariaDB version 10.0.21 which is an August release whereas XAMPP stack is the February release 10.0.17 maybe someone caught the corrupted table issue and Apache Friends thought 10.0.17 was stable. MariaDB notes the Release Status of 10.0.17 is stable.

Still trying to teach myself how to take 10.0.17 out of my stack and replace it with 10.0.21 which appears not to have the corrupted table problem.
